packages/composables/src/useCustomerOrders.ts

```diff
@@ -13,11 +13,22 @@ export type UseCustomerOrdersReturn = {
    *
    * In order to change a page with additional parameters please use `loadOrders` method.
    */
-  changeCurrentPage(pageNumber: number | string): Promise<void>;
+  changeCurrentPage(
+    pageNumber: number | string,
+    limit: number | string,
+  ): Promise<void>;
   /**
    * Fetches the orders list and assigns the result to the `orders` property
    */
   loadOrders(parameters?: Schemas["Criteria"]): Promise<void>;
+  /**
+   * Current page number
+   */
+  getCurrentPage: ComputedRef<number>;
```

Comment:
it's computed property, not getter function, we should rename it
```suggestion
  currentPage: ComputedRef<number>;
```